src/components/Profile/Profile.jsx

```diff
@@ -60,6 +60,10 @@ export const Profile = ({
     resetFetchedMember();
   }, []);
 
+  useEffect(() => {
+    resetFetchedMember();
+  }, [window.location.href]);
```

Comment:
As you are using hooks, you can use `useParams` hook to get the exact url param to be used as the dependency